job-service-db/src/main/resources/procedures/upsertJobPolicy.sql

```diff
@@ -0,0 +1,37 @@
+--
+-- Copyright 2016-2021 Micro Focus or one of its affiliates.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+/*
+ *  Name: internal_upsert_job_policy
+ *
+ *  Description:
+ *  Inserts or updates the policy for a specific job.
+ */
+CREATE OR REPLACE FUNCTION internal_upsert_job_policy(
+    in_partition_id VARCHAR(40),
+    in_job_id VARCHAR(58)/*,
+    in_status job_status,
+    in_reference_date VARCHAR(58),
+    in_duration INT*/
+)
+RETURNS VOID
+LANGUAGE plpgsql
+AS $$
+BEGIN
+
+    RAISE NOTICE 'Hello world';
```

Comment:
Resubmit this for review with an explanation of what parts of it you want reviewed and what should be ignored.

Reply:
My bad, I have worked so far on the policy validation and that file is for the next step: persisting it (at that point, I just wanted to see if I was reaching out). I will rework the PR to be worthy of being reviewed, then resubmit